coorx/linear.py

```diff
@@ -38,26 +38,76 @@ def _imap(self, coords):
         return coords
 
     def as_affine(self):
-        return AffineTransform(matrix=np.eye(self.dims[0]), offset=np.zeros(self.dims[0]))
+        return AffineTransform(
+            matrix=np.eye(self.dims[0]), offset=np.zeros(self.dims[0]), from_cs=self.systems[0], to_cs=self.systems[1]
+        )
 
     @property
     def full_matrix(self):
-        return np.eye(self.dims[0]+1)
+        return np.eye(self.dims[0] + 1)
```

Comment:
Wondering if full_matrix should only be a property of AffineTransform..

Reply:
it's a useful shorthand for `as_affine().full_matrix`, and does have a few small optimizations in some subclasses. `as_affine()` will raise NotImplemented for the non-linears, which will be true with with or without this method. what's the concern?

Reply:
maybe we get a LinearTransform baseclass. I'll ticket that.